docs/jobs/openings/exec-assistant.md

```diff
@@ -0,0 +1,37 @@
+## Executive Assistant opening @ Aragon (Zug, Switzerland)
+
+Aragon is an open source, decentralized project [fighting for the freedom](https://www.youtube.com/watch?v=AqjIWmiAidw) to organize. We believe that decentralized organizations can solve the world's worst problems.
+
+We’re looking for an Executive Assistant to support the Aragon founders with organization, coordination and time optimization. The ideal candidate must not only be someone highly efficient, with love for time self-management and adaptive to fast-paced environments, but also curious and eager to learn about decentralization.
```

Comment:
> eager to learn about decentralization.

How about `eager to learn about the decentralization and open-source movements.` ?

Reply:
I think decentralization implicitly involves open source. So I'd leave it as it is.